intrahospital_api/loader.py

```diff
@@ -49,85 +45,46 @@
 from elcid import models as emodels
 from opal.models import Patient
 from elcid.utils import timing
-from intrahospital_api import get_api
 from intrahospital_api.exceptions import BatchLoadError
 from intrahospital_api.constants import EXTERNAL_SYSTEM
 from intrahospital_api import update_demographics
-from intrahospital_api import update_lab_tests
+from intrahospital_api.services.lab_tests import service as lab_tests
 from intrahospital_api import logger
-
-api = get_api()
+from intrahospital_api import get_api
 
 
 @timing
-def initial_load():
-    models.InitialPatientLoad.objects.all().delete()
-    models.BatchPatientLoad.objects.all().delete()
-    batch = models.BatchPatientLoad()
-    batch.start()
+def initial_load(remaining=False):
+    """
+    Runs an initial load.
```

Comment:
This isn't a great docstring comment - it assumes I know what running an  initial load is, when I would do it etc...